app/views/connections/_connection.html.haml

```diff
@@ -0,0 +1,11 @@
+- index = connection_counter + 1
+%tr(class="connection_#{index}")
+  %td
+    .pull-left
+      %img(src="#{gravatar_image_string(connection.guide)}")
+      = link_to "#{connection.guide_name}", user_path(connection.guide)
+  %td.extra-padding #{connection.guide_queue_size}
```

Comment:
Rather than using brackets and interpolation, you can use it staring with `=`

`= connection.guide_queue_size`